src/sentry/locale/catalogs.json

```diff
@@ -1,16 +1,48 @@
 {
   "supported_locales": [
-    "bg", 
-    "cs", 
-    "de", 
-    "en", 
-    "es", 
-    "fi", 
-    "fr", 
-    "it", 
-    "ja", 
-    "ru", 
-    "zh_CN", 
+    "ach",
+    "af",
```

Comment:
should we also update the file https://github.com/getsentry/sentry/blob/master/static/app/data/languages.tsx?